versions-maven-plugin/src/it/it-dynamic-versioning-scm-tag-0-commits-0-tags/invoker.properties

```diff
@@ -0,0 +1,2 @@
+invoker.goals=${project.groupId}:${project.artifactId}:${project.version}:set-version-from-scm-tag
+invoker.buildResult = failure
```

Comment:
missing empty line at the and of file

Reply:
For this file only or also the others with :no_entry: ?

My settings in VS Code seems to cause this. Does not `mvn spotless:apply` format this correctly?